oauth2/src/main/java/com/okta/spring/boot/oauth/env/OktaOAuth2PropertiesMappingEnvironmentPostProcessor.java

```diff
@@ -158,18 +161,14 @@ private PropertySource oktaRedirectUriPropertySource(Environment environment) {
     }
 
     private PropertySource oktaStaticDiscoveryPropertySource(Environment environment) {
-        String oauth2Suffix = Optional.ofNullable(environment.getProperty("okta.oauth2.issuer"))
-            .filter(issuer -> issuer.contains("/oauth2")) // Check if URL is a Custom Authorization Server
-            .map(s -> "") // if so, leave URL as is
-            .orElse("/oauth2"); // for the Okta Org Authorization Server, /oauth2 needs to be appended
```

Comment:
This was mixing eager property initialization with delayed (which may be required for spring cloud config, or similar property resolution)